File: financial/Models/DelicatessenOrderProduct.cs
using System.ComponentModel.DataAnnotations.Schema;
namespace Models
{
    [Table("DelicatessenOrderProduct")]
    public class DelicatessenOrderProduct : BaseEntity
    {
        public int DelicatessenOrderId { get; set; }
        public int DelicatessenProductId { get; set; }
        public decimal Value { get; set; }
            public decimal Quantity { get; set; }

        [NotMapped]
        public virtual DelicatessenOrder DelicatessenOrder { get; set; }
        [NotMapped]
        public virtual DelicatessenProduct DelicatessenProduct { get; set; }

    }
}

File: android/app/src/main/java/com/br/app/deliverydelicatessen/model/DelicatessenOrderProduct.kt
package com.br.app.deliverydelicatessen.model

data class DelicatessenOrderProduct(
    var delicatessenOrderId: Int = 0,
    var delicatessenProductId: Int = 0,
    var id: Int = 0,
    var value: Double = 0.00,
    var quantity: Double = 0.00,
    var delicatessenProduct: DelicatessenProduct? = null,
) {
}